drivers/i2c/i2c_mcux_flexcomm.c

```diff
@@ -25,6 +25,7 @@ LOG_MODULE_REGISTER(mcux_flexcomm);
 
 #include "i2c-priv.h"
 
+#include <zephyr/pm/device.h>
 #define I2C_TRANSFER_TIMEOUT_MSEC                                                                  \
 	COND_CODE_0(CONFIG_I2C_NXP_TRANSFER_TIMEOUT, (K_FOREVER),                                  \
```

Comment:
I find it strange that we do not have to block off entering low power modes during I2C transfers.